docs/reference/operations-guide/federated-operation.md

```diff
@@ -0,0 +1,730 @@
+            },
+            "MakeModel": {
+            "source": "id.vehicle.string",
+            "destination": "id.manufacturer.string",
+            "directed": "true"
+            }
+        },
+        "entities": {
+            "Vehicle": {
+            "description": "Entity representing a vehicle vertex",
+            "vertex": "id.vehicle.string",
+            "properties": {
+                "type": "property.string"
+            },
+            "aggregate": false
+            },
+            "Manufacturer": {
+            "description": "Entity representing a manufacturer vertex",
+            "vertex": "id.manufacturer.string",
+            "properties": {
+                "name": "property.string"
+            },
+            "aggregate": false
+            },
+            "Job": {
+            "description": "Entity representing a job vertex",
+            "vertex": "id.job.string",
+            "properties": {
+                "type": "property.string"
+            },
+            "aggregate": false
+            },
+            "Person": {
+            "description": "Entity representing a person vertex",
+            "vertex": "id.person.string",
+            "properties": {
+                "name": "property.string"
+            },
+            "aggregate": false
+            },
+            "Place": {
+            "description": "Entity representing a place vertex",
+            "vertex": "id.place.string",
+            "properties": {
+                "name": "property.string"
+            },
+            "aggregate": false
+            }
+        },
+        "types": {
+            "id.person.string": {
+            "description": "A basic type to hold the string id of a person entity",
+            "class": "java.lang.String"
+            },
+            "id.place.string": {
+            "description": "A basic type to hold the string id of a place entity",
+            "class": "java.lang.String"
+            },
+            "id.job.string": {
+            "description": "A basic type to hold the string id of a job entity",
+            "class": "java.lang.String"
+            },
+            "property.string": {
+            "description": "A type to hold string properties of entities",
+            "class": "java.lang.String"
+            },
+            "true": {
+            "class": "java.lang.Boolean",
+            "validateFunctions": [
+                {
+                "class": "uk.gov.gchq.koryphe.impl.predicate.IsTrue"
+                }
+            ]
+            },
+            "id.manufacturer.string": {
+            "description": "A basic type to hold the string id of a manufacturer entity",
+            "class": "java.lang.String"
+            },
+            "id.vehicle.string": {
+            "description": "A basic type to hold the string id of a vehicle entity",
+            "class": "java.lang.String"
+            }
+        }
+    }
+    ```
+
+#### ConcatenateMergeFunction
```

Comment:
Might be good to mention that with this you can get duplicate results. As seen in the traits example query.